src/phases/replication/startup/StartupPhasesTemplate.yml

```diff
@@ -0,0 +1,237 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: |
+  Common definitions to support the workloads in replication/startup.
+
+DefaultNumPhases: &default_max_phase 13
+
+Document: &Doc
+  t: {^RandomInt: {min: 0, max: 10}}
+  w: {^RandomInt: {distribution: geometric, p: 0.1}}
+  x: 0
+  y: {^RandomInt: {min: 0, max: 1000}}
+  z: {^RandomInt: {min: 0, max: 2147483647}}  # This is for max int for low probability of conflicts
+  int0: &int {^RandomInt: {distribution: binomial, t: 200, p: 0.5}}
+  int1: *int
+  int2: *int
+  int3: *int
+  int4: *int
+  int5: *int
+  int6: *int
+  int7: *int
+  int8: *int
+  int9: *int
+  intArray:
+  - 1000
+  - 2000
+  - 3000
+  - 4000
+  - 5000
+  - 6000
+  - 7000
+  - 8000
+  - 9000
+  string0: &string {^RandomString: {length: {^RandomInt: {min: 5, max: 15}}}}
+  string1: *string
+  string2: *string
+  string3: *string
+  string4: *string
+  string5: *string
+  string6: *string
+  string7: *string
+  string8: *string
+  string9: *string
+  stringShort: {^RandomString: {length: 1}}
+  stringLong: {^RandomString: {length: 100}}
+  padding: {^FastRandomString: {length: {^Parameter: {Name: "approxDocumentSize", Default: 100}}}}
```

Comment:
Wont this create padding equal to the "approxDocumentSize", and thus create documents that are 2x the approxDocumentSize?

Reply:
Yes this is the variable that derives the document size, so the document should be approximately that variable size, I am not sure why 2x ?